amqp/method_framing.py

```diff
@@ -157,6 +157,10 @@ def frame_writer(connection, transport,
                 offset += 8 + framelen
 
                 framelen = len(body)
+
+                if isinstance(body, str):
+                    body = body.encode()
+
```

Comment:
At least we want a separate branch for Python 3 here, so this behavior doesn't leak into our beloved Python 2.